src/features/ITGovernance/Requester/TaskList/GovTaskGrbMeeting/index.tsx

```diff
@@ -283,6 +285,18 @@ const GovTaskGrbMeeting = ({
               </>
             )}
 
+          {
+            /** Discussions card */
+            // Only render if review is active
+            (grbMeetingStatus === ITGovGRBStatus.AWAITING_DECISION ||
+              grbMeetingStatus === ITGovGRBStatus.REVIEW_IN_PROGRESS) && (
+              <RequesterDiscussionsCard
+                systemIntakeId={id}
+                grbMeetingStatus={grbMeetingStatus}
+              />
+            )
+          }
+
           <div className="margin-top-2 display-flex flex-align-center">
             <UswdsReactLink
```

Comment:
the Prepare for the GRB Review button does not match figma:

![59530](https://github.com/user-attachments/assets/b49d5f5f-c825-4e33-8db5-c23d7c6a7d06)


![98715](https://github.com/user-attachments/assets/6ebb72fe-8a5f-4c22-ad23-2624b131a7d0)

Reply:
It looks like the button is wrong for a few different statuses. I'll put up a NOREF to fix this since it's unrelated to the discussions work. I want to go ahead and get this merged asap so I can continue with the next requester discussions ticket.